addons/website/controllers/main.py

```diff
@@ -1,16 +1,17 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
1st commit message:
Maybe add that this for the `Font Family` theme options so we know from the commit message that it improves these options :wink: